File: exercicios/1066/1066.py
a = [];
for i in range(5):
    n = int(input());
    a.append(int(n));

par = 0;
imp = 0;
pos = 0;
neg = 0;
for j in range(5):
    
    if (a[j] % 2) == 0:
        par = par + 1;

    if (a[j] % 2) == 1:
        imp = imp + 1;

    if a[j] > 0:
        pos = pos + 1;
    
    if a[j] < 0:
        neg = neg + 1;

print(par, 'valor(es) par(es)')
print(imp, 'valor(es) impar(es)')
print(pos, 'valor(es) positivo(s)')
print(neg, 'valor(es) negativo(s)')

File: exercicios/JavaScript/1066/1066.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var imp =0; par = 0, pos = 0, neg = 0;

for(let i=0; i<5; i++){
    var valor = parseInt(lines.shift());

    if(valor > 0){
        pos += 1;
    }else if(valor < 0){
        neg += 1;
    }
    if(valor%2 === 0){
        par += 1;
    }else if( valor%2 !== 0){
        imp += 1;
    }
}

console.log(par + " valor(es) par(es)\n"
            + imp + " valor(es) impar(es)\n"
            + pos + " valor(es) positivo(s)\n"
            + neg + " valor(es) negativo(s)"
);